modules/search/instant-search/lib/api.js

```diff
@@ -67,6 +127,7 @@ export function search( { aggregations, filter, query, resultFormat, siteId, sor
 			highlight_fields = [ 'title', 'content', 'comments' ];
 			fields = [
 				'date',
+				'modified',
```

Comment:
```suggestion
```

We shouldn't need this right? These are only the fields that get displayed. Was this just debugging code?

Reply:
Correct! Removed.